assets/src/components/map.tsx

```diff
@@ -412,6 +412,10 @@ const EventAdder = ({
       }
     },
 
+    popupopen: (e) => setTimeout(() => (e.popup.options.autoPan = false), 100),
```

Comment:
This is great! Thanks for linking that GH issue - definitely not intuitive to me that the `keepInView` wouldn't have done the trick!

question(non-blocking):  I wonder if using the [autopanstart](https://leafletjs.com/reference.html#map-autopanstart) event would remove the need to use a timeout (though the name is kind of confusing since it doesn't imply it is a popup-only event)

Reply:
So I searched the Leaflet repo and the only references to `autopanstart` I found were popup-related, but when I tried using it I got a runtime error because `e.popup` wasn't defined - I haven't dug in further to figure out exactly why but it doesn't look like it's quite a drop-in replacement.

Reply:
Okay, thanks for trying it anyway!

Reply:
A tangential but non-blocking question (maybe for design) - I wonder if auto-panning to the opened tooltip should disable auto-centering on the map? Otherwise if there is a pan to the fully opened tooltip, there might be another pan that moves the tooltip out of view when the vehicle's position is updated.

Reply:
Oh, that is a good question. I would say it probably should but I can check with design tomorrow.